spec/generateRequestObj.spec.js

```diff
@@ -92,17 +211,28 @@ describe('generateRequestObj', function () {
                         type: 'string',
                         minLength: 1
                     },
-                    data: {
-                        type: 'object'
-                    },
+                    data: getOpenrtbSchema(),
                     callbackId: {
                         type: 'string',
                         minLength: 1
+                    },
+                    networkParamOverrides: {
+                    	type: 'object',
+                    	properties: {
+                    		method: {
+                    			type: 'string',
+                    			eq: 'POST'
+                    		},
+                    		contentType: {
```

Comment:
Please remove the contenType validator as it is longer needed as it was removed in the main file.